content/posts/newsletter/2018-11-30.md

```diff
@@ -0,0 +1,102 @@
+---
+title: The XMPP Newsletter, 30 November 2018
+date: 2018-11-30 00:00
+author: jcbrand
+blog_id: newsletter 
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter.
+
+If you have an article, tutorial or blog post you'd like us to include in the
+newsletter, please [submit it on the XMPP wiki](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## News
+
+Paul Schaub has written about a [QR-code generator for OMEMO](https://blog.jabberhead.tk/2018/11/03/qr-code-generator-for-omemo/)
+which he has been working on. It encodes the Jabber-ID of the owner as well as any OMEMO
+fingerprints they choose to include.
+
+JC Brand has written about the [recent XMPP sprint held in Dusseldorf](https://opkode.com/blog/xmpp-sprint-dusseldorf/).
+Featuring sushi, ramen, whiskey, protocol discussions, translations, bugfixes, documentation, releases, hacks and coding.
+
+The [Monal Android and Mac client now shows a simple introduction to
+XMPP](https://monal.im/blog/the-welcome-screen/) with
+lovely illustrations of Peter rabbit, as he enters and escapes Mr McGregor's garden.
+There are also new [empty state screens](https://monal.im/blog/the-empty-state/) with similar illustrations.
+
+In the Netherlands, a chat service called IronChat, with an Android client which
+is apparently a fork of [Conversations](https://conversations.im/), was 
+compromised when the police took control of the centralized server being used
+and started a man-in-the-middle attack on the conversations. It seems the
+service was very popular with criminals, which is why the police got involved,
+and various arrests were made. [The original article in Dutch](https://nos.nl/artikel/2258309-beveiliging-door-politie-gekraakte-cryptofoons-was-twijfelachtig.html)
+and here's a [Google machine-translated version](https://translate.google.com/translate?hl=&sl=nl&tl=en&u=https%3A%2F%2Fnos.nl%2Fartikel%2F2258309-beveiliging-door-politie-gekraakte-cryptofoons-was-twijfelachtig.html).
```

Comment:
The casual reader might deduce from this that XMPP is unsafe to use / easy to crack. I'd love to see some emphasis on the fact that these guys were not up-to-date, using old and error-prone OTR end-to-end encryption, and using XMPP in a non-federated manner, which pretty much goes against the premise of providing a non-single-point-of-failure secure service.

Reply:
OTR wasn't cracked, their server was. How it was cracked we don't know because the Police didn't divulge any details.

Reply:
I'm not sure we can emphasize those things without knowing the details. Most likely their server got cracked in a way completely unrelated to XMPP.

In any case, the linked to article has more details.

Reply:
My main concern is that in the current wording, casual readers might think that it's a good idea to avoid XMPP.

Reply:
Initial attempt at a different approach, feel free to use in whole, in part or not at all:

```suggestion
In the Netherlands a commercial chat service based on XMPP and a modified version of
the open-source Xabber client for Android was compromised when the Dutch police gained
access to the server and implemented a man-in-the-middle attack on encrypted conversations
that used Off-The-Record (OTR) encryption.

OTR and other end-to-end encryption technologies such as OMEMO only guarantee end-to-end
security if you are able to verify the fingerprints of your contacts' encryption keys. In
this case it appears users did not verify or ignored warnings, and this opened the door to
a man-in-the-middle attack by anyone with access to the server.

The attack in this instance was for law enforcement. However it serves to highlight the
importance of designing user interfaces that encourage good security practices, and educating
users. It also highlights problems with centralized services - all IronChat users were on a
single XMPP server, which meant a single place to compromise and access all IronChat
conversations). A federated network does not have a single point through which all messages
pass.
```

Reply:
You mention Xabber, I've read Conversations before. Are you sure either way?

"all IronChat users were" -> "all IronChat users appear to have been" (unless you know more than us :P ), but otherwise, +1.

Reply:
@guusdk My source for the Xabber claim is the Xabber author himself, but no, I cannot personally say one way or the other (having never used IronChat or seen the source).

and fine with "appear to have been".

Reply:
@mwild1 Good summary thanks. I've used it.

Reply:
@guusdk It was a centralized service, so all users were on one server.